pkg/handler/rpms.go

```diff
@@ -215,6 +215,7 @@ func (rh *RpmHandler) detectRpmsPresence(c echo.Context) error {
 // @Param        search query string false "Term to filter and retrieve items that match the specified search criteria. Search term can include name."
 // @Param        type query string false "A comma separated list of types to control api response. Type can include `security`, `enhancement`, `bugfix`, and `other`."
 // @Param        severity query string false "A comma separated list of severities to control api response. Severity can include `Important`, `Critical`, `Moderate`, `Low`, and `Unknown`."
+// @Param		 sort_by query string false "Sort the response based on specific parameters. Sort criteria can include `issued_date`, `updated_date`, `type`, and `severity`."
```

Comment:
👀 is that a tab?

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExemZ6NzE1ZGVvN3YzaHhpaWwwbGRndDVqMmhyd3NzMGswOWRwbGs0dyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/LROKNqr26NG48/giphy.gif)

Reply:
my IDE does strange things sometimes T_T will fix! thanks for catching that